data-store/pb/data_store.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option go_package = "github.com/Veermove/zpi/store";
```

Comment:
Shouldn't it be `github.com/Veermove/zpi/data-store` to match the directory name?